keyboards/qck75/v1/info.json

```
@@ -0,0 +1,132 @@
+{                                                                                                                         
+    "manufacturer": "Andrian",                                                                                            
+    "keyboard_name": "QCK75-XT",                                                                                          
+    "maintainer": "PoringH",                                                                                              
+    "bootloader": "stm32-dfu",                                                                                            
+    "diode_direction": "COL2ROW",                                                                                         
+    "encoder": {                                                                                                          
+        "rotary": [                                                                                                       
+            {"pin_a": "C0", "pin_b": "C1", "resolution": 2}                                                               
+        ]                                                                                                                 
+    },                                                                                                                    
+    "features": {                                                                                                         
+        "bootmagic": true,                                                                                                
+        "command": false,                                                                                                 
+        "console": false,                                                                                                 
+        "encoder": true,                                                                                                  
+        "extrakey": true,                                                                                                 
+        "mousekey": true,
+        "nkro", true,
+        "oled", true,
```

Sorry, messed up the suggestion
```suggestion
        "nkro": true,
        "oled": true,
```

Formatting the info.json still needs to be done.